web/modules/custom/vote_utility/translations/zh-hans.po

```diff
@@ -248,3 +251,51 @@ msgstr "sábado"
 
 msgid "Sunday"
 msgstr "星期日"
+
+msgid "U.S. government information and services"
+msgstr "美国政府提供的信息和服务"
+
+#aria-label for election links
+msgid "External election resources"
+msgstr "选举链接"
+
+#footer
+msgid "Visit our partner site to find more information about voting in the United States."
+msgstr "访问我们合作伙伴的网站 —— USA.gov——来了解在美国投票的更多信息。"
+
+msgid "Close"
+msgstr "关闭"
+
+msgid "U.S. flag"
+msgstr "美国国旗"
+
+msgid "Vote.gov is made possible in partnership with "
+msgstr "Vote.gov的创建得益于 "
+
+msgid "USA.gov logo"
+msgstr "USA.gov标识"
+
+msgid "Menu"
+msgstr "菜单"
+
+msgid "Breadcrumb"
+msgstr "面包屑导航"
+
+msgid "Primary navigation"
+msgstr "主导航"
+
+msgid "Search component"
+msgstr "搜索组件"
+
+msgid "Introduction"
+msgstr "简介"
+
+msgid "Register to vote selector tool"
+msgstr "登记投票选择器工具"
+
+msgid "States and territories list"
+msgstr "州和属地建议清单"
+
+msgid "Select your state or territory"
+msgstr "州和属地建议清单"
+
```

Comment:
Missing "States and territories suggestion list" (tab 42)

Reply:
I believe we've not been adding the FVAP content since the hardcoded cards are being removed and some of the translations were missing these strings - I added the 1 that we have for this language but there is 1 missing on the spreadsheet.